servers/navigation/navigation_utilities.h

```diff
@@ -69,10 +69,10 @@ struct PathQueryParameters {
 };
 
 struct PathQueryResult {
-	Vector<Vector3> path;
-	Vector<int32_t> path_types;
+	PackedVector3Array path;
+	PackedInt32Array path_types;
 	TypedArray<RID> path_rids;
-	Vector<int64_t> path_owner_ids;
+	PackedInt64Array path_owner_ids;
```

Comment:
Why are we changing the server internal struct to Variant boilerplate here?

Reply:
There isn't any change to the underlying type, these are just typedefs in variant.h (https://github.com/godotengine/godot/blob/ed85a2c8ceae048c2b29bc6ccf6f53a9c70ec811/core/variant/variant.h#L70)

These changes were needed in the Gdextension PR due to an issue with the CPP binder where it failed to bind Vector<int64_t> and others correctly, but did handle the variant typed fine. I decided it didn't hurt to include them here.